app/models/delay_henka/scheduled_change.rb

```diff
@@ -12,20 +12,25 @@ class ScheduledChange < ApplicationRecord
 
     validates :submitted_by_id, :attribute_name, presence: true
     validates :state, inclusion: { in: STATES.values }
+    before_save :set_default_schedule_at
```

Comment:
this should only happen on create, correct?

Reply:
> this should only happen on create, correct?

yes